modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -1312,27 +1315,19 @@
 				Wraps float [code]value[/code] between [code]min[/code] and [code]max[/code].
 				Usable for creating loop-alike behavior or infinite surfaces.
 				[codeblock]
-				# a is 0.5
-				a = wrapf(10.5, 0.0, 10.0)
-				[/codeblock]
-				[codeblock]
-				# a is 9.5
-				a = wrapf(-0.5, 0.0, 10.0)
-				[/codeblock]
```

Comment:
Why remove those examples? Users are always asking for more examples.

Reply:
These examples aren't relevant for `wrapf`, because these are cases where it's better to use `fposmod`.